### ISSUE TITLE
CI test fails with jnumpy

### ISSUE THREAD

**Issue by songjhaha** [Suzhou-Tongyuan/jnumpy#8]
```
ERROR: LoadError: ArgumentError: Package TyPython [9c4566a2-237d-4c69-9a5e-9d27b7d0881b] is required but does not seem to be installed:
 - Run `Pkg.instantiate()` to install all recorded dependencies.
```

looks like `init_project` fails when writing TyPython dependencies to `Project.toml`
https://github.com/Suzhou-Tongyuan/jnumpy/blob/a3179656be5b721ea6c483bee8b049ad9677baa3/jnumpy/init.py#L157-L159

**Comment by thautwarm** [comment]
Sometimes a package is listed in dependencies but it might not be correctly instantiated. We need to distinguish this case and call `Pkg.develop` accordingly.

**Comment by thautwarm** [comment]
fixed by #9
